packages/ember-cli-fastboot/index.js

```diff
@@ -184,7 +184,8 @@ module.exports = {
     this._processAddons(this.project.addons, fastbootTrees);
     // check the parent containing the fastboot directory
     const projectFastbootPath = path.join(this.project.root, 'fastboot');
-    if (this.existsSync(projectFastbootPath)) {
+    // ignore the project's fastboot folder if we are an addon, as that is already handled above
+    if (!this.project.isEmberCLIAddon() && this.existsSync(projectFastbootPath)) {
```

Comment:
@simonihmig it would be great to add a test for this. Seems like we will need to add a `test-package` app that consumes an addon that uses `ember-cli-fastboot` 

#805 is going to change this repos structure over the next week. We may need to rebase this PR then we can get it merged.